hoomd/device.py

```diff
@@ -54,7 +40,8 @@ def __init__(self, communicator, notice_level, msg_file, shared_msg_file):
             self._comm = communicator
 
         # c++ messenger object
-        self.cpp_msg = _create_messenger(self.comm.cpp_mpi_conf, notice_level, msg_file, shared_msg_file)
+        self.cpp_msg = _create_messenger(self.comm.cpp_mpi_conf, notice_level,
```

Comment:
`cpp_msg` needs to be underscored since it is private as public classes inherit from `_Device`.